app/controllers/users_controller.rb

```diff
@@ -46,6 +42,14 @@ def show
 			set_vars_from_recipe(current_user.recipes.where(:type => "PlanRecipe")[2], "dinner")
 		end
 	end
+
+	def set_alert
+		if @all_recipes.nil?
+			return "API limit reached, please try again in one minute!"
+		else
```

Comment:
is this else case specifically for when the api returns a hash, but the hash is empty?

Reply:
yes